examples/inundation_qld.ipynb

```diff
@@ -9,30 +24,65 @@
     "import fiona\n",
     "import numpy as np\n",
     "import pandas as pd\n",
+    "import os, sys, urllib\n",
+    "\n",
+    "#add path to module so that we can re-run the notebook\n",
+    "sys.path.append('../../wit_tooling/')\n",
```

Comment:
please don't do this we could do something like `!pip install git+git://github.com/GeoscienceAustralia/wit_tooling` at the very beginning of the notebook if this notebook is not distributed with the package

Reply:
thanks! done